api/turing/batch/runner/ensembling.go

```diff
@@ -0,0 +1,309 @@
+package batchrunner
```

Comment:
Runner is a generic interface where you can run any BatchJobRunner implementations. Perhaps this would be better?

• turing/batch/ensembling
  • controller.go
  • controller_test.go
  • runner.go
• turing/batch/runner
  • base.go

Reply:
This makes sense

Reply:
This is a little tricky to mock due to cyclic dependencies. I have made a note in the test file saying that we have to use the `--inpackage` flag for mocking.